packages/components/src/internal/components/samples/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from '../base/models/User';
+import { App } from '../../../index';
+
+export function getOmittedSampleTypeColumns(user: User, omitCols = null): string[] {
```

Comment:
`omitCols` should be typed as `string[]`. Additionally, it doesn't need to be assigned to a default value of `null` as it will default to `undefined` if made optional.

```ts
export function getOmittedSampleTypeColumns(user: User, omitCols?: string[]): string[] {
```

Reply:
Done!